### PR TITLE
added tags for python and misc folder

### PR DESCRIPTION
**Pull request by linamnt**

### CODE REVIEW

File: lessons/misc/ruby-intro/lesson.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 layout: page
 title: 'Intro to Ruby'
 visible: true
+tag:
+  - misc
```
Comment (mbonsma) [inline lessons/misc/ruby-intro/lesson.md:6]: Why is `misc` lower-case and everything else is upper-case?

Comment (linamnt) [reply lessons/misc/ruby-intro/lesson.md:6]: that's a good point, I'll change this. I'm not sure why I did that

File: lessons/misc/jekyll-ghpages/lesson.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 layout: page
 title: 'Web design with Jekyll/GitHub Pages'
 visible: true
+tag:
+  - misc
+  - Beginner
```
Comment (lwjohnst86) [inline lessons/misc/jekyll-ghpages/lesson.md]: `s/B/b`

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]

File: lessons/misc/jekyll-ghpages/lesson.md (hunk 2 above)
Comment (lwjohnst86) [reply lessons/misc/jekyll-ghpages/lesson.md]: Damn, Maddy already mentioned it

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**